packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.jsx

```diff
@@ -1,29 +1,58 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Button, ThemedScrollbars, Carousel } from '@deriv/components';
+import { Button, ThemedScrollbars, Carousel, ButtonToggle } from '@deriv/components';
 import { isMobile } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import TradeCategories from 'Assets/Trading/Categories/trade-categories.jsx';
 import TradeCategoriesGIF from 'Assets/Trading/Categories/trade-categories-gif.jsx';
 import { getContractTypes } from '../../../../Helpers/contract-type';
+import ContractTypeGlossary from './contract-type-glossary.jsx';
+import classNames from 'classnames';
 
 const Info = ({ handleNavigationClick, handleSelect, initial_index, item, list }) => {
+    const [carousel_index, setCarouselIndex] = React.useState(initial_index);
+    const [selected_tab, setSelectedTab] = React.useState('description');
     const contract_types = getContractTypes(list, item).filter(i => i.value !== 'rise_fall_equal');
+    const onClickGlossary = () => setSelectedTab('glossary');
     const cards = contract_types.map((type, idx) => {
         return (
-            <div key={idx} className='contract-type-info__card'>
-                <div className='contract-type-info__gif'>
-                    <TradeCategoriesGIF category={type.value} />
-                </div>
-                <ThemedScrollbars
-                    className='contract-type-info__scrollbars'
-                    height={isMobile() ? '' : '300px'}
-                    autohide={false}
-                >
-                    <div className='contract-type-info__content'>
-                        <TradeCategories category={type.value} />
-                    </div>
-                </ThemedScrollbars>
+            <div
+                key={idx}
+                className={classNames('contract-type-info__card', {
+                    'contract-type-info__card--has-toggle-buttons': carousel_index !== 8 && carousel_index !== 1,
+                })}
+            >
+                {selected_tab === 'description' ? (
+                    <React.Fragment>
+                        <div
+                            className={classNames('contract-type-info__gif', {
+                                'contract-type-info__gif--has-toggle-buttons':
+                                    carousel_index !== 8 && carousel_index !== 1,
+                            })}
+                        >
+                            <TradeCategoriesGIF category={type.value} />
+                        </div>
+                        <ThemedScrollbars
```

Comment:
Need clarification from designers as in Figma prototype picture of trade type is also scrollable. In that case `<ThemedScrollbars />` should be lifted up in order to include all.

Reply:
picture is now part of the scroll window 👍